universal-login-react/src/ui/TopUp/Fiat/FiatPaymentMethods.tsx

```diff
@@ -32,11 +33,11 @@ const FiatProviderTopUp = ({topUpProvider, providerLogo, setPaymentMethod, curre
   </TopUpRadio>
 );
 
-export const FiatPaymentMethods = ({selectedCountry, supportService, paymentMethod, setPaymentMethod, logoColor = 'white'}: FiatPaymentMethodsProps) => {
-  const rampLogo = getOnRampProviderLogo('ramp', logoColor);
-  const safelloLogo = getOnRampProviderLogo('safello', logoColor);
-  const wyreLogo = getOnRampProviderLogo('wyre', logoColor);
-
+export const FiatPaymentMethods = ({selectedCountry, supportService, paymentMethod, setPaymentMethod, logoColor}: FiatPaymentMethodsProps) => {
+  const getLogoColorByTheme = (useThemeName() === 'default') ? 'white' : 'black';
```

Comment:
`getLogoColorByTheme` is not a function so it shouldn't start with `get`. Another thing is that we are repeating `logoColor || getLogoColorByTheme` but we can do it this way:

```ts
const logoColorByTheme = (useThemeName() === 'default') ? 'white' : 'black';
const logoColor = givenLogoColor || getLogoColorByTheme;
```

Reply:
#NextPR